### PR TITLE
[IMP] marketing automation: workflow activities

### PR DESCRIPTION
**Pull request by samueljlieber**
Task: [#3699212](https://www.odoo.com/web#id=3699212&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form)

This PR improves the content of the `workflow_activities.rst` doc for 17.0 updates and expanded details for creating activities.

`marketing_automation.rst` was also updated to include references to `workflow_activities.rst` and a small correction.

`email_communication/faq.rst` was updated to include a custom anchor used in `workflow_activities.rst`.

### CODE REVIEW

**Comment by samueljlieber** [comment]
Hi @ksc-odoo this doc is ready for a peer/content review whenever you get a chance, thank you! :)

File: content/applications/marketing/marketing_automation/workflow_activities.rst [hunk 1]
```diff
@@ -2,265 +2,335 @@
...
 
-.. image:: workflow_activities/create-edit-sms-option.png
+.. image:: workflow_activities/sms-activity-type.png
    :align: center
    :alt: The create and edit email drop-down option on create activities pop-up window.
 
-Doing so reveals a :guilabel:`Create Marketing Template` pop-up window. In that pop-up window,
-proceed to create and configure the new SMS template.
+Once the SMS template is configured, click :guilabel:`Save & Close` to save the activity and return
+to the :guilabel:`Create Activities` modal and continue to configure the :ref:`trigger
```
Comment (ksc-odoo) [inline content/applications/marketing/marketing_automation/workflow_activities.rst:165]:
```suggestion
to the :guilabel:`Create Activities` modal, in order to configure the :ref:`trigger
```

File: content/applications/marketing/marketing_automation/workflow_activities.rst [hunk 2]
```diff
@@ -2,265 +2,335 @@
...
-  the sent SMS in the workflow has not been clicked by the recipient, after it's been opened.
-- :guilabel:`SMS: bounced`: the activity will be triggered at the previously-configured time if the
-  sent SMS in the workflow has been bounced back for any reason.
+   :alt: The trigger field group on Create Activities modal.
 
-Expiry duration
----------------
+To set an execution delay for the activity from when the :ref:`trigger type
+<marketing_automation/trigger-type>` occurs, type a whole number in the *interval number* input
+(e.g., `2` is valid, `0` is also valid, and `1.5` is not).
```
Comment (ksc-odoo) [inline content/applications/marketing/marketing_automation/workflow_activities.rst:186]:
```suggestion
(e.g. `2` is valid, `0` is also valid, and `1.5` is not).
```

File: content/applications/marketing/marketing_automation/workflow_activities.rst [hunk 3]
```diff
@@ -2,265 +2,335 @@
...
+
+      - :guilabel:`Add Another Activity`: to be executed after the parent activity.
+      - :guilabel:`Clicked`: a link in the SMS was clicked by the participant.
+      - :guilabel:`Not Clicked`: a link in the SMS was **not** clicked by the participant.
+      - :guilabel:`Bounced`: the SMS has bounced.
+
+Once a trigger type is selected, the :guilabel:`Create Activities` modal opens to configure the
+child activity. The process is the same as :ref:`creating a new activity
+<marketing_automation/activities>`, with the exception that the :guilabel:`Trigger` field is
+pre-filled with the selected trigger type and the and :guilabel:`Activity` field has the parent
```
Comment (ksc-odoo) [inline content/applications/marketing/marketing_automation/workflow_activities.rst:330]:
```suggestion
pre-filled with the selected trigger type, and the :guilabel:`Activity` field has the parent
```

**Review by ksc-odoo — CHANGES_REQUESTED** [review]
Hey @samueljlieber -- just finished my Final Review of this doc. Great job. Super detailed and in-depth! Once you implement the necessary changes, feel free to tag me agianf or another quick look. Thanks! 👍 

REMEMBER: fix all modal mentions and add 'app' when necessary to menuselections

File: content/applications/marketing/marketing_automation/workflow_activities.rst [hunk 4]
```diff
@@ -2,265 +2,335 @@
...
-   :alt: A workflow activity template pop-up window in Odoo Marketing Automation.
+Activities
+==========
+
+Activities are the methods of communication or server actions, organized in a workflow, that are
+executed within a campaign; they are the building blocks of the campaign's workflow.
+
+A new activity can be added to the workflow on a campaign form by selecting an existing campaign or
+:ref:`creating a new campaign <marketing_automation/campaigns>` from the :menuselection:`Marketing
+Automation --> Campaigns` dashboard, then clicking the :guilabel:`Add new activity` button. Doing so
```
[the remark this answers is not in the document]
Comment (StraubCreative) [reply content/applications/marketing/marketing_automation/workflow_activities.rst:26]: I don't recall this being a strict guideline? IIRC this is up to the writer whether or not they want to do this.

[the remark this answers is not in the document]
Comment (samueljlieber) [reply content/applications/marketing/marketing_automation/workflow_activities.rst:26]: That is how I recall [the conversation](https://docs.google.com/document/d/1laiWC9ow0IjXPuQp5h8L_Tj98HbE8ymGn9v-FFOwlYM/edit) as well, that it is up to the writer. However, I do like being precise so I will change every instance to include 'app' like KSC is suggesting :)

File: content/applications/marketing/marketing_automation/workflow_activities.rst [hunk 5]
```diff
@@ -2,265 +2,335 @@
...
-Email activity
-~~~~~~~~~~~~~~
+Email
+~~~~~
 
-If :guilabel:`Email` is the :guilabel:`Activity Type`, the option to choose a premade/pre-configured
-:guilabel:`Mail Template` becomes available. A mail template can also be created on-the-fly, as
-well.
+If :guilabel:`Email` is selected as the :guilabel:`Activity Type`, the option to :guilabel:`Pick or
+Create a Template...` in the :guilabel:`Mail Template` field is available.
```
[the remark this answers is not in the document]
Comment (samueljlieber) [reply content/applications/marketing/marketing_automation/workflow_activities.rst:66]: I totally agree, I got carried away with the create flow 😄 I will modify this to be exact 👍

**Comment by samueljlieber** [comment]
Thank you so much @ksc-odoo for your review, your suggestions were very helpful and I'm happy with the direction this doc is heading. This PR should be ready for another look when you have a chance. Thanks again!

File: content/applications/marketing/marketing_automation/workflow_activities.rst [hunk 6]
```diff
@@ -2,265 +2,336 @@
...
-There is also the option to add a *child activity* by clicking :guilabel:`Add child activity`,
-located at the bottom of each activity block in the :guilabel:`Workflow` section of a marketing
-campaign form.
+Activities that are connected to, and triggered by, another activity are known as, *child
+activities*.
 
-Child activities are sub-activities that are connected to and triggered by the activity above it,
-which is known as its *parent activity*. A child activity is easy to recognize, as its slightly
-indented beneath its parent activity.
+The activity that triggers a child activity is known as it's *parent activity*.
```
Comment (ksc-odoo) [inline content/applications/marketing/marketing_automation/workflow_activities.rst:277]:
```suggestion
The activity that triggers a child activity is known as its *parent activity*.
```

**Review by ksc-odoo — APPROVED** [review]
Looking good, @samueljlieber 👍 just one, super minor adjustment needed. But, once that's taken care of, feel free to move this to the next stage! Nicely done 👏

**Comment by StraubCreative** [comment]
Remaining CR implemented on [6a45381](https://github.com/odoo/documentation/pull/8527/commits/6a4538139c684c697b336613a75a97dea72efac0).

**Review by StraubCreative — APPROVED** [review]
Looks good, thanks @samueljlieber 
@robodoo r+

**Pull request closed without merging**